### PR TITLE
SERV-370 req.japi.type fix bugs

### PR DESCRIPTION
**Pull request by chepil**

### CODE REVIEW

File: testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.jst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 JSIGHT 0.3
 
-TYPE @ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789___1_-_-_1 // Long description with different symbols... tabs:			ascii: !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~ spec:¹²³$‰↑∞←→—≠€®™ѣѵіѳ′[]≈§°£₽„“”‘’×©↓−«»…´“„•ẞˇ¢·¸¨‘⌀⌘˘˚ѢѴІѲ″{}±–〉〈¿ˆ¼⅓½¡ Cyrillic: йцукенгшщзФЫВАПРОЛДЖЭ Chinese: 你好世界！
```
Comment (andrew-platonov) [inline testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.jst:3]: Why did you  change the test? Just leave it as it is and put the whole file as "*.todo"

File: testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.openapi [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "title": "",
+    "version": ""
+  },
+  "components": {
+    "schemas": {
+      "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789___1_-_-_1": {
+        "additionalProperties": false,
+        "properties": {},
+        "type": "object",
+        "description": "Short description. TODO: Fix bug with long description, task SERV-374"
```
Comment (andrew-platonov) [inline testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.openapi:13]: The same comment, as before. Why did you change the test? It is ok if it fails until the bug is fixed.

**Review by andrew-platonov — CHANGES_REQUESTED** [review]

File: testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.jst (hunk 1 above)
Comment (chepil) [reply testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.jst]: fixed, but '#' inside single line annotation will stop tests after uncomment todo

File: testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.openapi (hunk 2 above)
Comment (chepil) [reply testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.openapi]: fixed, but '#' inside single line annotation will stop tests after uncomment todo

**Pull request merged**